hail/src/main/scala/is/hail/expr/ir/Optimize.scala

```diff
@@ -3,9 +3,10 @@ package is.hail.expr.ir
 import is.hail.utils._
 
 object Optimize {
-  private def optimize(ir0: BaseIR, noisy: Boolean, canGenerateLiterals: Boolean): BaseIR = {
+  private def optimize(ir0: BaseIR, noisy: Boolean, canGenerateLiterals: Boolean, context: Option[String]): BaseIR = {
+    val contextStr = context.map(s => s" ($s)").getOrElse("")
     if (noisy)
-      log.info("optimize: before:\n" + Pretty(ir0, elideLiterals = true))
+      log.info(s"optimize$context: before: IR size ${ IRSize(ir0) }: \n" + Pretty(ir0, elideLiterals = true))
```

Comment:
Do you want a space before context?

Reply:
no, the `Some` case adds a space.

Reply:
Oh, so `context` should be `contextStr`.

Reply:
d'oh. good catch!

Reply:
done